### PR TITLE
completed week2 nodeJs assignment

### PR DESCRIPTION
**Pull request by shreyansh-khatri**
Pull Request Questionnaire

1) Briefly describe your change.
Completed NodeJs week2 assignments.

2) What parts of the application are affected by your change?
My changes only affected the week2 branch of this repository.

3) How did you test your change?
I tested various parts such as validation, different inputs and some other edge cases.

### CODE REVIEW

**Review by RitikaGarg-ITT — APPROVED** [review]

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    // "noImplicitOverride": true,                       /* Ensure overriding members in derived classes are marked with an override modifier. */
+    // "noPropertyAccessFromIndexSignature": true,       /* Enforces using indexed accessors for keys declared using an indexed type. */
+    // "allowUnusedLabels": true,                        /* Disable error reporting for unused labels. */
+    // "allowUnreachableCode": true,                     /* Disable error reporting for unreachable code. */
+
+    /* Completeness */
+    // "skipDefaultLibCheck": true,                      /* Skip type checking .d.ts files that are included with TypeScript. */
+    "skipLibCheck": true                                 /* Skip type checking all .d.ts files. */
+  }
+}
```
Comment (choudhary-keshav) [inline tsconfig.json:10]: remove this commented code from tsconfig file!

**Review by choudhary-keshav — CHANGES_REQUESTED** [review]

**Review by aditya-s-19 — APPROVED** [review]

**Review by choudhary-keshav — APPROVED** [review]

**Review by shreegovinda — APPROVED** [review]

**Review by anusreesubash — APPROVED** [review]

**Review by Harrrrry — APPROVED** [review]
